src/uct/cuda/gdr_copy/gdr_copy_md.c

```diff
@@ -25,6 +25,12 @@ static ucs_config_field_t uct_gdr_copy_md_config_table[] = {
     {"", "", NULL,
      ucs_offsetof(uct_gdr_copy_md_config_t, super), UCS_CONFIG_TYPE_TABLE(uct_md_config_table)},
 
+    {"SINGLE", "y", "Use single instance per application",
```

Comment:
why is this option needed?

Reply:
to disable this feature and effectively restore the original behavior if problems arise that make this patch suspect